packages/cardpay-reward-programs/cardpay_reward_programs/rules/dummy_rule.py

```diff
@@ -0,0 +1,86 @@
+import duckdb
+import pandas as pd
+from cardpay_reward_programs.rule import Rule
+
+
+class DummyRule(Rule):
+    """
+    This rules rewards acounts with
```

Comment:
missing description. Its not clear to me immediately what the dummy rule is

Reply:
deleted